Microsoft.Azure.CosmosRepository/src/Dtos/PagedResult.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Azure.CosmosRepository.Dtos
+{
+    /// <inheritdoc />
+    public class PagedResult<T> : ListResult<T>
+    {
+        /// <inheritdoc />
+        public long TotalCount { get; set; }
```

Comment:
```suggestion
        public long TotalCount { get; internal set; }
```

Do not let consumers set this value, we'll manage it internally.